app/templates/briefs/view_response_result.html

```diff
@@ -0,0 +1,52 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Your response to ‘{{ brief.title }}’ - Digital Marketplace{% endblock %}
+
+{% block main_content %}
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <header class="page-heading-smaller">
```

Comment:
Because the base page I copied from to make this one doesn't use it either ;)
I'll fix it up.